evm/src/main/java/org/hyperledger/besu/evm/fluent/SimpleAccount.java

```diff
@@ -47,6 +47,7 @@ public class SimpleAccount implements EvmAccount, MutableAccount {
   private Supplier<Hash> codeHash =
       Suppliers.memoize(() -> code == null ? Hash.EMPTY : Hash.hash(code));
   private final Map<UInt256, UInt256> storage = new HashMap<>();
+  private final Map<UInt256, UInt256> transientStorage = new HashMap<>();
```

Comment:
It's a HashMap here - why is it a TreeMap in the other impl?

Reply:
<Responded to the other comment>